cpp/src/aztec/benchmark/honk_bench/honk.bench.cpp

```diff
@@ -40,14 +33,18 @@ void generate_test_plonk_circuit(auto& composer, size_t log_num_gates)
 void create_unrolled_prover_bench(State& state) noexcept
 {
     for (auto _ : state) {
```

Comment:
Maybe `auto&`?  Could these state objects be heavy?

Reply:
Its a good point but 1) They don't do this in any of the docs, and 2) I think (at least for us) `state` is quite a light object so probably ok to leave for now.

Reply:
OK, works for me.